set.mm

```diff
@@ -776556,6 +776840,676 @@ separated by (open) neighborhoods.  See ~ sepnsepo for the equivalence
+      DVHVREWDVGVQVCVDVEVFFUNUOUPUQURUSUTUTABEIJKVAVB $.
+
+    $( A thin category is a category in which all hom-sets has cardinality less
+       than or equal to the cardinality of ` 1o ` .  (Contributed by Zhi Wang,
+       17-Sep-2024.) $)
+    isthinc2 $p |- ( C e. ThinCat <->
+            ( C e. Cat /\ A. x e. B A. y e. B ( x H y ) ~<_ 1o ) ) $=
+      ( vf cthinc wcel ccat cv co wmo wral wa c1o cdom wbr isthinc modom2 bitri
+      2ralbii anbi2i ) DIJDKJZHLALBLEMZJHNZBCOACOZPUEUFQRSZBCOACOZPABCDHEFGTUHU
+      JUEUGUIABCCHUFUAUCUDUB $.
+
+    $( A thin category is a category in which, given a pair of objects ` x `
+       and ` y ` and any two morphisms ` f , g ` from ` x ` to ` y ` , the
+       morphisms are equal.  (Contributed by Zhi Wang, 17-Sep-2024.) $)
+    isthinc3 $p |- ( C e. ThinCat <-> ( C e. Cat /\
+      A. x e. B A. y e. B A. f e. ( x H y ) A. g e. ( x H y ) f = g ) ) $=
+      ( cthinc wcel ccat cv co wmo wral wa weq isthinc moel 2ralbii anbi2i
+      bitri ) DJKDLKZEMAMBMGNZKEOZBCPACPZQUDEFRFUEPEUEPZBCPACPZQABCDEGHISUGUIUD
+      UFUHABCCEFUETUAUBUC $.
+  $}
+
+  ${
+    $d B w y z $.  $d B x y z $.  $d F k l $.  $d G l $.  $d H f k w $.
+    $d H k l $.  $d H f x y z $.  $d X f k w $.  $d X k l $.  $d X f w z $.
+    $d Y k l $.  $d Y k w $.  $d k ph $.  $d ph z $.
+    isthincd2lem1.1 $e |- ( ph -> X e. B ) $.
+    isthincd2lem1.2 $e |- ( ph -> Y e. B ) $.
+    isthincd2lem1.3 $e |- ( ph -> F e. ( X H Y ) ) $.
+    isthincd2lem1.4 $e |- ( ph -> G e. ( X H Y ) ) $.
+    ${
+      isthincd2lem1.5 $e |- ( ph ->
+                              A. x e. B A. y e. B E* f f e. ( x H y ) ) $.
+      $( Lemma for ~ isthincd2 and ~ thincum .  (Contributed by Zhi Wang,
+         17-Sep-2024.) $)
+      isthincd2lem1 $p |- ( ph -> F = G ) $=
+        ( vk cv wceq wral wcel vl vz vw wmo oveq1 eleq2d mobidv oveq2 cbvral2vw
+        co sylib nfv eleq1w cbvmow syl5bb wa eqidd rspc2vd mpd moel eqeq1 eqeq2
+        ) APQZUAQZRZUAIJHUJZSPVFSZFGRZAVCVFTZPUDZVGAEQZUBQZUCQZHUJZTZEUDZUCDSUB
+        DSZVJAVKBQZCQZHUJZTZEUDZCDSBDSVQOWBVPVKVLVSHUJZTZEUDBCUBUCDDVRVLRZWAWDE
+        WEVTWCVKVRVLVSHUEUFUGVSVMRZWDVOEWFWCVNVKVSVMVLHUHUFUGUIUKAVJVKIVMHUJZTZ
+        EUDZVPUBUCIJDDDVLIRZVOWHEWJVNWGVKVLIVMHUEUFUGWIVCWGTZPUDVMJRZVJWHWKEPWH
+        PULWKEULEPWGUMUNWLWKVIPWLWGVFVCVMJIHUHUFUGUOKAWJUPDUQLURUSPUAVFUTUKAVHF
+        VDRVEPUAFGVFVFVFVCFVDVAVDGFVBMAVCFRUPVFUQNURUS $.
+    $}
+
+    ${
+      $d B f $.  $d C f x y $.
+      thincum.b $e |- B = ( Base ` C ) $.
+      thincum.h $e |- H = ( Hom ` C ) $.
+      thincum.c $e |- ( ph -> C e. ThinCat ) $.
+      $( Morphisms in the same hom-set are identical.  ("um" stands for unique
+         morphism.)  (Contributed by Zhi Wang, 17-Sep-2024.) $)
+      thincum $p |- ( ph -> F = G ) $=
+        ( vx vy vf wcel cv cthinc co wmo wral isthinc simprbi syl isthincd2lem1
+        ccat ) APQBRDEFGHIJKLACUASZRTPTQTFUBSRUCQBUDPBUDZOUJCUISUKPQBCRFMNUEUFU
+        GUH $.
```

Comment:
I think the usual abbreviation for uniqueness is "eu" (this form corresponds to eu6)

Reply:
In that case, it would rather correspond to "at most one", so `mo` (but the quantifier `E*` does not appear in the statement...). Maybe `thincatmo` ?

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4220/commits/cef1d97ba1bd83c4d782cc92e8d30cf4160d3fb8

Reply:
I think `mo` should be fine cause it is implied... Anyway I already had a `mo` so I renamed to `mo2`.